iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -259,6 +263,23 @@ TEST_F(PoshRuntime_test, GetMiddlewarePublisherIsSuccessful)
     EXPECT_EQ(publisherOptions.historyCapacity, publisherPort->m_chunkSenderData.m_historyCapacity);
 }
 
+TEST_F(PoshRuntime_test, GetMiddlewarePublisherMaxCapacity)
+{
+    // arrange
+    iox::popo::PublisherOptions publisherOptions;
+    publisherOptions.historyCapacity = 17U;
```

Comment:
Please set  ``publisherOptions.historyCapacity = iox::MAX_PUBLISHER_HISTORY + 1U``. This would make the intention clearer and the code more robust against changes of the max capacity.